### PR TITLE
Polish Proxy policy docs and explain how to use path management parameters — Connect LunchBadger/general#425

### LINKED ISSUE #180: document with example how proxy appends req.url to forwarded url
there is a description for that.
http://www.express-gateway.io/docs/policies/proxy

`example: API endpoint - "http://api.foobar.com/api" proxied to a service endpoint defined as "http://internal.api.lan:8080/" will have "/api" appended to it to become "http://internal/api/lan:8080/api"`

more examples should be added

### PR DESCRIPTION
**Pull request by XVincentX**
Docs for new proxy policy strip parameter and clarifications around proxy path parameters

Connect #180 
Closes #180

### CODE REVIEW

File: docs/policies/proxy.md [hunk 1]
```diff
@@ -78,9 +70,11 @@ pipelines:
 * `toProxy`:
   - true/false, passes the absolute URL as the `path` (useful for proxying to proxies)
 * `prependPath`:
-  - true/false, Default: true - specify whether you want to prepend the target's path to the proxy path
+  - true/false, Default: true - specify whether you want to prepend the target's path to the proxy path. See more details below.
 * `ignorePath`:
-  - true/false, Default: false - specify whether you want to ignore the proxy path of the incoming request (note: you will have to append / manually if required).
+  - true/false, Default: false - specify whether you want to ignore the proxy path of the incoming request. See more details below.
+* `stripPath`
+  - true/false, Default: false — specify whether you want to strip the `apiEndpoint` path from the final URL. See more details below.
```
Comment (altsang) [inline docs/policies/proxy.md:77]: maybe - `final URL of the target path` to make it more clear and to have it match what you have above?  think we should define what the proxy path is versus the target path.  They sound the same to me?

File: docs/policies/proxy.md [hunk 2]
```diff
@@ -134,9 +128,72 @@ pipelines:
...
+
+According to the options you will get different results:
+
+```yml
+prependPath:  false
+ignorePath:   false
+stripPath:    false
+```
+
+Resulting upstream service path: `/public/api/billing/byName?name=Clark`
```
Comment (altsang) [inline docs/policies/proxy.md:156]: think we to normalize here and get back to "proxy path versus target path" and not introduce another term - "service path"

same for the mentions below?

**Review by altsang — APPROVED** [review]
this looks great with more examples

**Comment by XVincentX** [comment]
I'll keep these in minds when merging and releasing. Thanks.

**Pull request merged**